test/s3_options.js

```diff
@@ -30,5 +30,11 @@ export default {
   cloudfrontInvalidateOptions: {
     DistributionId: CLOUDFRONT_DISTRIBUTION_ID,
     Items: ['/*']
-  }
+  },
+
+  indexOptions: {
```

Comment:
I'm not sure what the new options do, what does setting s3 to true vs cloudfront to true mean?